### PR TITLE
i18n path update

### PR DESCRIPTION
**Pull request by RishabhKodes**
Updated the path for the i18n

Fixes Externalization issues #97

### CODE REVIEW

**Comment by bipuladh** [comment]
There are instances of `ceph-storage-plugin` used in the codebase. Can you please remove them and run the i18n command again?

File: packages/odf/components/system-list/odf-system-list.tsx [hunk 1]
```diff
@@ -178,42 +178,42 @@ const SystemTableRow: React.FC<RowFunctionArgs<StorageSystemKind, CustomData>> =
   };
 
 const StorageSystemList: React.FC<StorageSystemListProps> = (props) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation('ceph-storage-plugin');
```
Comment (bipuladh) [inline packages/odf/components/system-list/odf-system-list.tsx:181]:
```suggestion
  const { t } = useTranslation('plugin__odf-console');
```

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx [hunk 2]
```diff
@@ -146,7 +146,7 @@ const AddSSCapacityModal: React.FC<AddSSCapacityModalProps> = ({
 };
 
 const AddCapacityModal = (props: AddCapacityModalProps) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation('ceph-storage-plugin');
```
Comment (bipuladh) [inline packages/odf/modals/add-capacity/add-capacity-modal.tsx:149]:
```suggestion
  const { t } = useTranslation('plugin__odf-console');
```

**Review by bipuladh — CHANGES_REQUESTED** [review]
https://github.com/red-hat-storage/odf-console/blob/master/packages/shared/status/statuses.tsx#L10-L15
Update it here too.

File: packages/shared/storage/pvc/pvs-available-capacity.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const PVsAvailableCapacity: React.FC<PVAvaialbleCapacityProps> = ({
   loaded,
   loadError,
 }) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation('plugin_odf-console');
```
Comment (bipuladh) [inline packages/shared/storage/pvc/pvs-available-capacity.tsx:22]:
```suggestion
  const { t } = useTranslation('plugin__odf-console');
```

File: packages/shared/status/statuses.tsx [hunk 4]
```diff
@@ -7,12 +7,12 @@ import { useTranslation } from 'react-i18next';
 import { StatusComponentProps } from './types';
 
 export const SuccessStatus: React.FC<StatusComponentProps> = (props) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation('plugin_odf-console');
```
Comment (bipuladh) [inline packages/shared/status/statuses.tsx:10]:
```suggestion
  const { t } = useTranslation('plugin__odf-console');
```

File: packages/odf/components/system-list/odf-system-list.tsx [hunk 5]
```diff
@@ -178,42 +178,42 @@ const SystemTableRow: React.FC<RowFunctionArgs<StorageSystemKind, CustomData>> =
   };
 
 const StorageSystemList: React.FC<StorageSystemListProps> = (props) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation('plugin_odf-console');
```
Comment (bipuladh) [inline packages/odf/components/system-list/odf-system-list.tsx:181]:
```suggestion
  const { t } = useTranslation('plugin__odf-console');
```

**Review by bipuladh — CHANGES_REQUESTED** [review]
Please run `yarn i18n` as well.

**Review by bipuladh — CHANGES_REQUESTED** [review]
You need to change the namespace(ns prop) of Trans as well at https://github.com/red-hat-storage/odf-console/blob/master/packages/odf/modals/add-capacity/add-capacity-modal.tsx#L324-L325

File: locales/en/console-shared.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (bipuladh) [inline locales/en/console-shared.json:1]: This file should not exist.

**Comment by RishabhKodes** [comment]
> You need to change the namespace(ns prop) of Trans as well at https://github.com/red-hat-storage/odf-console/blob/master/packages/odf/modals/add-capacity/add-capacity-modal.tsx#L324-L325

The trans is already updated in the latest commit.

File: locales/en/ceph-storage-plugin.json [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (bipuladh) [inline locales/en/ceph-storage-plugin.json:1]: This file should not exist at all.

**Review by bipuladh — CHANGES_REQUESTED** [review]

**Review by bipuladh — COMMENTED** [review]
There are some strings starting with `public~` can we change that as well?

**Comment by RishabhKodes** [comment]
> There are some strings starting with `public~` can we change that as well?

Changing `public~` to `plugin__odf-console`?

**Comment by bipuladh** [comment]
> > There are some strings starting with `public~` can we change that as well?
> 
> Changing `public~` to `plugin__odf-console~`?

No remove `public~` and pass `plugin__odf-console` to `t` funciton.

File: packages/odf/constants/common.ts [hunk 8]
```diff
@@ -14,12 +14,12 @@ export enum defaultRequestSize {
 
 export const requestedCapacityTooltip = (t: TFunction) =>
     t(
-        'ceph-storage-plugin~The amount of capacity that would be dynamically allocated on the selected StorageClass.',
+        'The amount of capacity that would be dynamically allocated on the selected StorageClass.',
```
Comment (bipuladh) [inline packages/odf/constants/common.ts:17]:
```suggestion
        'plugin__odf-console~The amount of capacity that would be dynamically allocated on the selected StorageClass.',
```

File: packages/odf/constants/common.ts [hunk 9]
```diff
@@ -14,12 +14,12 @@ export enum defaultRequestSize {
...
     );
 
 export const storageClassTooltip = (t: TFunction) =>
     t(
-        'ceph-storage-plugin~The StorageClass used by OpenShift Data Foundation to write its data and metadata.',
+        'The StorageClass used by OpenShift Data Foundation to write its data and metadata.',
```
Comment (bipuladh) [inline packages/odf/constants/common.ts:22]:
```suggestion
        'plugin__odf-console~The StorageClass used by OpenShift Data Foundation to write its data and metadata.',
```

File: packages/odf/components/system-list/utils.ts [hunk 10]
```diff
@@ -8,7 +8,7 @@ import { StorageSystemKind } from '../../types';
 const addStorage = (kind, resource: StorageSystemKind, _, customData) => {
     const t: TFunction = customData?.tFunction;
     return {
-        labelKey: t('ceph-storage-plugin~Add Capacity'),
+        labelKey: t('Add Capacity'),
```
Comment (bipuladh) [inline packages/odf/components/system-list/utils.ts:11]:
```suggestion
        labelKey: t('plugin__odf-console~Add Capacity'),
```

**Review by bipuladh — CHANGES_REQUESTED** [review]

**Review by bipuladh — APPROVED** [review]
Please squash the commits.

**Pull request merged**